packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -89,6 +89,8 @@ import {
   version
 } from '../../util/helpers';
 
+type MutationBatch = api.Write[];
```

Comment:
FWIW I'm not super wild about this name. It's an array of `Writes` not "mutations". It also makes things more confusing since we have members like `earlyWrites` that now point to mutation batches.

Note that `api.Write` is the proto while `Mutation` is our internal type. It's worth it not to confuse these.

Reply:
I struggled to name this, and the best I could come up with is either MutationBatch or WriteRequest (since I don't want earlyWrites to be a WriteList[]). I ended up going with WriteRequest - and removing the type altogether since we can just use api.WriteRequest.